index.html

```diff
@@ -0,0 +1,229 @@
+            and<br> experience! If you like what you see and<br>
+            have a project you need coded, don’t<br> hestiate to
+            contact me.</p>
+        <div class="connect"><a href="#" class="connect">LET'S CONNECT</a></div>
+        <div class="contacts">
+            <ul class="social-icon">
+                <li>
+                    <a href="#" class="link"><img src="assets/images/Vector (2).svg" alt="icon5">
+                            </a>
+                </li>
+                <li><a href="#" class="link"><img src="assets/images/Linkedin.svg" alt="icon4"></a></li>
+                <li><a href="#" class="link"><img src="assets/images/Vector (3).svg" alt="icon3"></a></li>
+                <li><a href="#" class="link"><img src="assets/images/Vector.svg" alt="icon2"></a>
+                    <li><a href="#" class="link">
+                        <img src="assets/images/Vector (1).svg" alt="icon1">
+                        </a>
+                    </li>
+                </ul>
+            </div>
+
+            <button class="press" type="button">Get Resume</button></div>
+
+            <div class="skills">
+                <p class="skill"> Languages<a class="arw"><img src="assets/images/arrow icon.png" alt="arrow"></a></p>
+                <ul class="languages">
+                    <li class="options"><img src="assets/images/Ellipse 1.png" alt="JS" class="tech-img"><a
+                            href="#" class="tech">JavaScript</a></li>
+                    <li class="options"> <img src="assets/images/Ellipse 2.png" alt="html" class="tech-img"><a
+                            href="#" class="tech">HTML5</a></li>
+                    <li class="options"><img src="assets/images/Ellipse 3.png" alt="css" class="tech-img"><a
+                            href="#" class="tech">CSS</a></li>
+                </ul>
+                <p class="arrow">Frameworks <a><img src="assets/images/Union.png" alt="arrow"></a></p>
+                <hr>
+                <p class="arrow">Skills <a><img src="assets/images/Union.png" alt="arrow"></a>
+                <hr>
+
+            </div>
+        </div>
+
+        
+    <div class="contact-form">
+         <div class="first-container" >
+             <h2>Contact me </h2>
+            <p class="para">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        </div>
+        <div class="formspree">
+         <form action="https://formspree.io/f/xdobrbok" method="post">
+             
+                <input type = "text" id="name"  name="user_name" maxlength = "30" placeholder="Enter Your name"  required>
+                <input type="email" id="email" name="user_email" placeholder="Enter your email" required>
+                <textarea name="message" id="msg" cols="25" rows="10" maxlength = "500" placeholder="Write your message here" required></textarea>
```

Comment:
I think you should use labels for these inputs.

Reply:
can you show a example?